include/openssl/ssl.h

```diff
@@ -2598,6 +2598,10 @@ int ERR_load_SSL_strings(void);
 # define SSL_R_X509_LIB                                   268
 # define SSL_R_X509_VERIFICATION_SETUP_PROBLEMS           269
 
+
+void DTLS_set_timer_cb(SSL *s, unsigned (*cb)(SSL *s, unsigned timer));
```

Comment:
do I have to create a completely new test, or can I extend an existing
one such as "dtlstest" ?